### PR TITLE
Pycon2016

### PR DESCRIPTION
**Pull request by Allard-Timothy**
New tests for catmath.

### CODE REVIEW

File: tests/test_catmath.py [hunk 1]
```diff
@@ -1,19 +1,39 @@
...
+import pytest
 
 
-def test__cat_years_to_hooman_years__middle_age__succeeds():
-    assert True
+@pytest.mark.parametrize(('tc', 'expct'), [
+    (6, 30),
+    (5, 25),
+    (4, 20),
+    (10, 50)
```
Comment (bobcatfish) [inline tests/test_catmath.py:9]: nice use of parameterization!

File: tests/test_catmath.py [hunk 2]
```diff
@@ -1,19 +1,39 @@
...
+])
+def test__cat_years_to_hooman_years__less_than_one_year__succeeds(tc, expct):
+    assert catmath.cat_years_to_hooman_years(tc) == expct
 
 
-def test__cat_years_to_hooman_years__0__returns_0():
-    assert True
+@pytest.mark.parametrize(('tc', 'expct'), [
+    (0, 0),
+])
```
Comment (bobcatfish) [inline tests/test_catmath.py:26]: i'm interested in why you decided to use paramaterization when there is only one case?

File: tests/test_catmath.py (hunk 1 above)
Comment (bobcatfish) [inline tests/test_catmath.py:9]: how did you decide what values to use?

File: tests/test_catmath.py (hunk 2 above)
Comment (Allard-Timothy) [inline tests/test_catmath.py:26]: I was on a roll 😺. But in all seriousness I try to implement the same pattern for my tests in a file.

File: tests/test_catmath.py (hunk 1 above)
Comment (Allard-Timothy) [inline tests/test_catmath.py:9]: In retrospect I don't think my values align with the function name, they should have been in the 30-50 year range(middle age).

**Pull request closed without merging**